odxtools/diaglayer.py

```diff
@@ -193,13 +244,9 @@ def services(self) -> NamedItemList[Union[DiagService, SingleEcuJob]]:
         return self._services
 
     @property
-    def data_object_properties(self) -> NamedItemList[DopBase]:
-        """All data object properties including inherited ones.
-        This attribute corresponds to all specializations of DOP-BASE
-        defined in the DIAG-DATA-DICTIONARY-SPEC of this diag layer as well as
-        in the DIAG-DATA-DICTIONARY-SPEC of any parent.
-        """
-        return self._data_object_properties
+    def diag_data_dictionary_spec(self) -> DiagDataDictionarySpec:
```

Comment:
property name too long, could be potentially shortened without issues

Reply:
I agree. the problem is that the ODX spec calls this aberration DIAG-DATA-DICTIONARY-SPEC, and IMO it would be quite confusing to use something else. We can probably think about moving all objects which currently it contains to the diaglayer, but that would either make writing PDX files pretty hard and it would potentially confuse people who are familiar with the standard...